.travis.yml

```diff
@@ -8,120 +8,147 @@ stages:
   - build
   - test
 
+templates:
+  pr-jdk8: &pr-jdk8
+    if: type = pull_request OR repo != scala/scala
+
+  push-jdk16: &push-jdk16
+    if: type = push AND repo = scala/scala
```

Comment:
So build on every merge (and tags - and sneaky commit pushes).  Which means 3 extra jobs every merge.  I wouldn't mind avoiding that overhead on PR merges and having this be a fixed daily +3 jobs, by having these build nightly instead.

Reply:
That's fine with me 👍